components/input.css

```diff
@@ -0,0 +1,26 @@
+@import url(/stylesheets/main.css);
+
+.input {
+  display: block;
+  padding: 0.5rem;
+  width: 15rem;
+  margin-top: 5px;
+}
+
+.focus-input {
+  border: 2px solid hsl(var(--secondary-color));
+}
+
```

Comment:
You can directly use var(--secondary-color) since you are not using hsla.

Reply:
but I have defined the variables as values of hsl only. please look into the color.css file to confirm

Reply:
> You can directly use var(--secondary-color) since you are not using hsla.

I have tested it in local that cannot be done because of the way variables are defined.